src/test/fuzz/util.h

```diff
@@ -214,4 +214,16 @@ NODISCARD inline bool ContainsSpentInput(const CTransaction& tx, const CCoinsVie
     return false;
 }
 
+// Returns a byte vector of size `length` regardless of the number of remaining bytes available
+// from the fuzzer. Pads with zero value bytes if needed to achieve a vector size of `length`.
```

Comment:
Sadly doxygen is disabled on the test directory, but for good karma I suggest to still use doxygen comments (see dev notes for example formattings)

Reply:
Fixed!